apps/test-suite/tests/Recording.js

```diff
@@ -75,8 +75,10 @@ export async function test(t) {
     let recordingObject = null;
 
     t.beforeEach(async () => {
-      const { status } = await Permissions.getAsync(Permissions.AUDIO_RECORDING);
-      t.expect(status).toEqual('granted');
+      if (Platform.OS !== 'web') {
```

Comment:
I don't understand, isn't the permission required on web as well?

Reply:
In hindsight I am not sure why this change is in here, sorry.

I have removed it, and the tests pass as they should (also tested without any pre-set permissions in my browser).